src/screens/ProfileScreen/elements/BottomSheetImage.js

```diff
@@ -26,7 +27,7 @@ const BottomSheetImage = React.forwardRef((props, ref) => (
             <MediaIcon width={16.67} height={16.67} fill={colors.black} />
             <Text style={styles.textCard}>Upload from library</Text>
           </View>
-          {props.isLoadingUpdateImageGalery ? (
+          {props.isLoadingUpdateImageGallery ? (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a typo in the prop name `isLoadingUpdateImageGallery`. It was previously `isLoadingUpdateImageGalery` with one 'l' in 'Gallery'. Make sure that this prop is correctly updated everywhere it's used.
```diff
- {props.isLoadingUpdateImageGallery ? (
+ {props.isLoadingUpdateImageGalery ? (
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->